my-app/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react'
+import logo from './logo.svg'
+import './App.css';
+
+class Board extends Component {
+  constructor() {
+    super();
+    this.size = 9;
+    this.state = {
+
+      // wIsNext: true,
+      board: this.createGameBoard(this.size),
+
+    };
+  }
+
+  createGameBoard(size){
+    let gameBoard = [];
+      for(let x = 0; x < size; x++){
+        gameBoard[x] = []
+        for (let y = 0; y < size; y++){
+          gameBoard[x][y] = ['Y:',y,',','X:',x]
+        }
+    }
+
+    return gameBoard
+  }
+
+  handleClick(x,y) {
+    const squares = this.state.squares.slice();
+
+    squares[x][y] = this.state.wIsNext ? 'W' : 'B';
+    this.setState({
+      squares: squares,
+      wIsNext: !this.state.wIsNext,
+    });
+  }
+
+
+  render() {
+
+    let rows = this.state.board.map(function(row) {
```

Comment:
This looks like a responsibility that is secondary to `render` - consider refactoring this into an appropriately named function in this component.